### PR TITLE
Simplify Install Task & Precompile Initializer

### PR DESCRIPTION
**Pull request by Ancez**
- moved the if to be done inline so now instead of adding 3 lines, it adds 1 line to the layout

### CODE REVIEW

**Comment by Ancez** [comment]
@kirillplatonov let me know what you think of these changes 🙂  I hope you are well

I also wonder whether we could somehow get the js include tag to not be required in the app layout as realistically we want it in all the layouts (my example being the devise layout) so we either should be looping over all the layouts and inserting it to all of these or somehow get this outside of the app layouts 🤔🤷‍♂️

File: lib/hotwire/livereload/engine.rb [hunk 1]
```diff
@@ -15,10 +15,8 @@ class Engine < ::Rails::Engine
         #{root}/app/helpers
       )
 
-      initializer "hotwire_livereload.assets" do
-        if Rails.application.config.respond_to?(:assets)
-          Rails.application.config.assets.precompile += %w( hotwire-livereload.js )
-        end
+      initializer 'hotwire_livereload.assets.precompile' do |app|
+        app.config.assets.precompile += %w( hotwire-livereload.js )
```
Comment (kirillplatonov) [inline lib/hotwire/livereload/engine.rb]: I think we still need to check if config responds to `assets` as the app might not use sprockets and use https://github.com/rails/propshaft instead.

**Comment by kirillplatonov** [comment]
@Ancez one-line installer looks good to me 🙂

As for alternatives for including a livereload tag in the layout - I'm not sure about that. We could probably add rack middleware or `after_action` in the controller that will modify output HTML and inject script tag automatically.

File: lib/hotwire/livereload/engine.rb (hunk 1 above)
Comment (Ancez) [reply lib/hotwire/livereload/engine.rb]: yep! Great call, I've updated the PR 🙌

this is something I've got to add to my gem too 😆  noticed it on one of the projects

File: lib/hotwire/livereload/engine.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ class Engine < ::Rails::Engine
         #{root}/app/helpers
       )
 
-      initializer "hotwire_livereload.assets" do
+      initializer 'hotwire_livereload.assets' do
```
Comment (Ancez) [inline lib/hotwire/livereload/engine.rb:18]: what're your thoughts on the usage of single quotes instead? I tend to stick to single quotes unless I need variables in the string

**Comment by kirillplatonov** [comment]
@Ancez thanks for the work on this PR. I have to close it as another PR having similar changes has been merged: https://github.com/kirillplatonov/hotwire-livereload/pull/22

**Pull request closed without merging**